newIDE/app/src/InstancesEditor/Grid.js

```diff
@@ -1,4 +1,17 @@
 import * as PIXI from 'pixi.js-legacy';
+import { type RGBColor } from '../UI/ColorField/ColorPicker';
+
+// Equal to #6868E8
+const DEFAULT_COLOR: RGBColor = {
+  r: 104,
+  g: 104,
+  b: 232,
+};
+
+const rgbColorToHex = (rgbColor: RGBColor) => {
+  const { r, g, b } = rgbColor;
+  return PIXI.utils.rgb2hex([r, g, b].map(c => c / 255));
+};
```

Comment:
Could you move this to `newIDE/app/src/Utils/ColorTransformer.js`, along with the other methods? Also try not to use `PIXI` inside this file, to make it independent from the rendering engine.